### PR TITLE
Add a mini dataset and fix the mobilebert accuracy on GPU

### PR DESCRIPTION
**Pull request by thaink**
The mini dataset contains 160 questions.
With the recent sync to the tensorflow.
The flag --host_cxxopt='--std=c++14' is currently required.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -4,10 +4,10 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "org_tensorflow",
-    sha256 = "3dd1e31116dd76a9276e3808da564b4723c732ac88107dc7aaf74191a33dbb0b",
-    strip_prefix = "tensorflow-2.3.0-rc0",
+    sha256 = "3fdba2ca9804d0fcefa0c0c8dff48c6b2f59201a9bf3fd0b359e6eed67ed5804",
```
Comment (terryheo) [inline WORKSPACE:7]: is this for GPU fix? otherwise how about tensorflow-2.3.0-rc1 ?

File: java/org/mlperf/inference/assets/tasks.pbtxt [hunk 2]
```diff
@@ -51,7 +51,7 @@ task {
   dataset {
     name: "Squad"
     type: SQUAD
-    path: "@assets/squad_eval.tfrecord"
+    path: "@assets/squad_eval_mini.tfrecord"
```
Comment (terryheo) [inline java/org/mlperf/inference/assets/tasks.pbtxt:54]: could you document how to use full evaluation?

File: WORKSPACE (hunk 1 above)
Comment (thaink) [reply WORKSPACE:7]: The GPU fix is only available yesterday. so It is not available in that version yet.

File: java/org/mlperf/inference/assets/tasks.pbtxt (hunk 2 above)
Comment (thaink) [reply java/org/mlperf/inference/assets/tasks.pbtxt:54]: Done. Added to /cpp/datasets/README.md

**Comment by terryheo** [comment]
BTW, squad_eval_mini.tfrecord is updated separately?

**Comment by thaink** [comment]
> BTW, squad_eval_mini.tfrecord is updated separately?

Right. It is not related to the accuracy bug on GPU.

**Comment by terryheo** [comment]
What I meant was your PR doesn't have squad_eval_mini.tfrecord.
Will you update it separately?

**Comment by thaink** [comment]
Could you check it again.
The java/org/mlperf/inference/assets/squad_eval_mini.tfrecord is added in this PR.

**Comment by terryheo** [comment]
Oh I see. There is some UI issue on showing that. Thanks for the confirmation.

**Pull request merged**